### PR TITLE
GEOPY-1534: Drillholes imported from .las files have no survey information - affects basic GA functionality

### PR DESCRIPTION
**Pull request by domfournier**
**GEOPY-1534 - Drillholes imported from .las files have no survey information - affects basic GA functionality**

### CODE REVIEW

File: las_geoh5/import_las.py [hunk 1]
```diff
@@ -373,23 +373,42 @@ def las_to_drillhole(
...
             translator=translator,
             logger=logger,
             collocation_tolerance=options.collocation_tolerance,
         )
-        ind = [drillhole.name == s.name.rstrip(".las") for s in survey]
-        if any(ind):
-            survey_path = survey[np.where(ind)[0][0]]
-            _ = add_survey(survey_path, drillhole, logger)
+
+    for drillhole in tqdm(drillhole_group.children, desc=""):
```
Comment (benk-mira) [inline las_geoh5/import_las.py:393]:
```suggestion
    for drillhole in tqdm(drillhole_group.children, desc="Attaching survey data."):

```

Comment (benk-mira) [reply las_geoh5/import_las.py:393]: ..or drop tqdm, it's already got one running for the data loop.

Comment (benk-mira) [inline las_geoh5/import_las.py:393]: This seems like it should all go inside add_survey.  Pass the whole list of surveys into it and resolve the from file vs. max depth inside?

**Review by benk-mira — CHANGES_REQUESTED** [review]

File: las_geoh5/import_las.py (hunk 1 above)
Comment (domfournier) [reply las_geoh5/import_las.py]: The check for single line survey becomes more complicated as the same drillhole could be hit multiple times depending on the file structure (duplicates). In which case i would need to do some comparison. It's another loop, but the logic for survey would only get hit once per hole here.

**Review by benk-mira — APPROVED** [review]

**Pull request merged**